README.md

```diff
@@ -4,10 +4,20 @@
 graphics, audio and network.
```

Comment:
Maybe this should be changed to mention that SFML.Net itself doesn't provide networking (i.e. we rely on .NET's own implementation)?

Reply:
> The network module is not provided in the SFML.Net binding as .NET provides superior networking support.